examples/pytorch/speech-recognition/run_speech_recognition_ctc.py

```diff
@@ -579,12 +632,24 @@ def remove_special_characters(batch):
         cache_dir=model_args.cache_dir,
         config=config,
         use_auth_token=data_args.use_auth_token,
+        ignore_mismatched_sizes=True,
```

Comment:
This is needed when instantiating from CTC checkpoints

Reply:
Good catch